.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Context
+
+AI/gen-ai-hub-sdk-js-backlog#ISSUENUMBER.
+
+Please provide a short description of what your change does and why it is needed.
```

Comment:
[req] I would put this line under a section `## Context` and remove `Feature Scope`.

Reply:
I am sorry, I did not understand. Why do you want to remove `Feature Scope` ?

Reply:
[pp] I honestly am not a fan of this. The scope of the ticket is already defined in the linked BLI and not everything is a feature. Unless I misunderstood what is meant here.
The context/description should be enough to describe overall changes or serve as a note to reviewers about what they should look out for. I would not restrict how one describes this.

Reply:
But why do you think there is a restriction here? It is just a reminder to add in the description right?
For smaller BLI's I guess people can just link the BLIs but for e.g API proposals or PRs that cover a BLI partially, I think this could be super helpful. 
Is it okay if we just make this optional depending on the BLI?

Reply:
Ah, I see that you made it optional :D.